clang/tools/clang-format/git-clang-format.bat

```diff
@@ -0,0 +1 @@
+py -3 git-clang-format %*
```

Comment:
What is `py`? I have no program called `py` on my machine (Windows with msys2), so you would possibly break that? Why not write `python` or `python3`?

Reply:
`py.exe` is the Windows python launcher. It comes with Python for Windows (not msys2 or cygwin where the shebang line would work). It is the Windows way of choosing which python version you want. You need to use this approach because the standard Python for Windows install always installs `python.exe` regardless of version because Windows generally doesn't version binaries.

[1] [Python for Windows docs](https://docs.python.org/3/using/windows.html#launcher)
[2] [Microsoft docs](https://learn.microsoft.com/en-us/windows/python/faqs#what-is-py-exe-)

I'm going to go take a long shower now to try and forget that I learned this much about Windows.

Reply:
@llvm-beanz This doesn't seem to work if `git-clang-format` is run from another directory:
```
C:\Users\Owen\llvm-project>clang\tools\clang-format\git-clang-format

C:\Users\Owen\llvm-project>py -3 git-clang-format
C:\Users\Owen\AppData\Local\Programs\Python\Python39\python.exe: can't open file 'C:\Users\Owen\llvm-project\git-clang-format': [Errno 2] No such file or directory

C:\Users\Owen\llvm-project>
```